File: angular/src/app/gwebsite/asset/dto/asset.dto.ts
import { ComboboxItemDto } from '@shared/service-proxies/service-proxies';

export class AssetDto{
    id: number;
    area: string;
    unitcode: string;
    areacode: string;
    transaction: string;
    assetcode: string;
    assetname: string;
    serinumber: string;
    originalprice: number;
    parentId: number | null;
}

export class GetAssetOutput{
    asset: AssetDto;
    assets: ComboboxItemDto[];
}

File: asset-management-api/src/GWebsite.AbpZeroTemplate.Application.Share/Dat_Assets/Dto/AssetDto.cs
using GWebsite.AbpZeroTemplate.Core.Models;
using System;
using Abp.Domain.Entities;
namespace GWebsite.AbpZeroTemplate.Application.Share.Assets.Dto
{
    /// <summary>
    /// <model cref="Asset"></model>
    /// </summary>
    public class AssetDto : Entity<int>
    {
        public String Area { get; set; }
        public String Unitcode { get; set; }
        public String Areacode { get; set; }
        public String Transaction { get; set; }
        public String Assetcode { get; set; }
        public String Assetname { get; set; }
        public String Serinumber { get; set; }
        public int Originalprice { get; set; }
        public int? ParentId { get; set; }
        public bool Status { get; set; }
    }
}
